src/safeds/exceptions/_data.py

```diff
@@ -23,8 +23,27 @@ def __init__(self, column_names: list[str]):
 class NonNumericColumnError(Exception):
     """Exception raised for trying to do numerical operations on a non-numerical column."""
 
-    def __init__(self, column_info: str) -> None:
-        super().__init__(f"Tried to do a numerical operation on one or multiple non numerical Columns: \n{column_info}")
+    def __init__(self, column_info: str, help_msg: str | None = None) -> None:
+        line_break = "\n"
```

Comment:
Variable could be left out to improve code readability.

Reply:
We did it this way so we only print the line break if we have a help message to improve the readability of the error message